### DISCUSSION TITLE
Missing import library in installed perl extensions

### DISCUSSION THREAD

**Discussion by sisyphus** [StrawberryPerl/Perl-Dist-Strawberry#128] [Q&A] [answered]
It used to be the case that when one built  and installed a perl extension (ie perl module that contained C and/or XS code) such as (eg) Glib, 2 of the files that were installed were:
1) the import lib, <code>perl/site/lib/auto/Glib/Glib.a</code>;
2) the dll. <code>perl/site/lib/auto/Glib/Glib.xs.dll</code>.

These days, the import lib is not installed - as it's *generally* not needed.
Does anyone here know how to restore the old behaviour of also installing the import lib ?
(AFAIK, ExtUtils::Depends expects the import lib to be there.)

Cheers,
Rob

**Comment by shawnlaffan** [comment]
I would assume that's an EUMM thing, but searching for ```.a``` files under the ```perl/vendor/lib/auto``` dir for Strawberry Perl 5.38.0 gives these results (using Ubuntu via WSL2):

```
find . -name '*.a' | grep -v share
./B/Hooks/OP/Check/Check.a
./B/Utils/Utils.a
./bareword/filehandles/filehandles.a
./multidimensional/multidimensional.a
```

Maybe something can be gleaned from those distributions?

**Comment by sisyphus** [comment, marked as answer]
Thanks - B::Hooks::OP::Check was the module I was trying to think of.
The difference is that SP has EU-Depends-0.8000, whereas I have EU-Depends-0.8001.
In version 0.8001, @kmx's hack that enabled things to work was deliberately removed because the "method is now obsolete, as well as causing problems."

Cheers,
Rob
